pkgs/development/libraries/x265/default.nix

```diff
@@ -1,7 +1,7 @@
 { stdenv, fetchurl, fetchpatch, cmake, nasm, numactl
 , numaSupport ? stdenv.hostPlatform.isLinux && (stdenv.hostPlatform.isx86 || stdenv.hostPlatform.isAarch64)  # Enabled by default on NUMA platforms
 , debugSupport ? false # Run-time sanity checks (debugging)
-, highbitdepthSupport ? false # false=8bits per channel, true=10/12bits per channel
+, highbitdepthSupport ? true # false=8bits per channel, true=10/12bits per channel
```

Comment:
Why is the default changed?

Reply:
10-bit video is widely available, and I don't see the downside of enabling this by default? Without this as the default it's pretty annoying to get an ffmpeg that can transcode HDR stuff.

Reply:
Sure, but do it in a separate commit. It's not needed for the update.